offer-manager-domain/src/main/groovy/life/qbic/business/offers/fetch/FetchOfferInput.groovy

```diff
@@ -0,0 +1,26 @@
+package life.qbic.business.offers.fetch
+
+import life.qbic.datamodel.dtos.business.OfferId
+
+/**
+ * Input interface for the {@link life.qbic.business.offers.fetch.FetchOffer} use case
+ *
+ * @since: 1.0.0
+ */
+interface FetchOfferInput {
+
+  /**
+   * Saves an offer in a (persistent) datasource.
```

Comment:
```suggestion
   * Saves an offer in a datasource.
```

Reply:
Ah, I just saw that this is actually not the case. It does not save anything

Reply:
You're right it's a remenant from copying the create offer documentation 🤦

Reply:
also called `copy & waste` :D